src/components/open-source.js

```diff
@@ -0,0 +1,72 @@
+import SectionHeader from "./section-header";
+import SectionInfo from "./section-info";
+import OpenSourceItem from "./open-source-item";
+const OpenSource = ({styles}) => {
+	return (
+		<div>
+			<SectionHeader>Open Source</SectionHeader>
+			<SectionInfo>
+				We make an effort to contribute back to the tools we use, and work to contribute tools
+				we find useful interally back to the community at large. Below you can find links to some
+				of our active open source projects.
+			</SectionInfo>
+			<div style={styles.gridContainer}>
+				<OpenSourceItem 
```

Comment:
would it be cleaner to have the defaultProps object have an array of meta and have a single function on the class that creates this list? as the list gets bigger would be nice to just have to update defaultProps?